src/components/Form2Preview.js

```diff
@@ -3,106 +3,11 @@ import './Form2Preview.css';
-                    </tr>
-                    <tr>
-                        <td style={{ fontWeight:'bold', verticalAlign:'top' }}>AND:</td>
-                        <td id="respondent-name" style={{ paddingLeft:'4px' }}>{this.props.data.respondents.map( (respondent, index) => {
-                            if (index === this.props.data.respondents.length - 1) {
-                                return respondent.name;
-                            } else {
-                                return respondent.name + ', ';
-                            }
-                        } ) }
-                        </td>
-                        <td style={{ textAlign:'right', fontWeight:'bold', paddingLeft:'70px', verticalAlign:'top' }}>Respondent{this.props.data.respondents.length > 1 ? 's' : '' }</td>
-                    </tr>
-                    </tbody></table>
-                    
-                    <h3 style={{ textAlign:'center' }}>NOTICE OF APPEARANCE</h3>                    
-                    <table width="100%"><tbody>
-                    <tr>
-                        <td>
-                        <strong>Enter an appearance on behalf of:</strong>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            { selectedRespondent.name }
-                        </td>
-                    </tr>
-                    <tr><td><strong>Address for service:</strong></td></tr>
-                    <tr>
-                        <td>
-                            {selectedRespondent.address ? selectedRespondent.address.addressLine1 : ''}
-                            &nbsp;
-                            {selectedRespondent.address ? selectedRespondent.address.addressLine2 : ''}
-                            
-                            <br/>
-                            {selectedRespondent.address ? selectedRespondent.address.city : ''}
-                            , {selectedRespondent.address ? selectedRespondent.address.province : ''}
-                            , {selectedRespondent.address ? selectedRespondent.address.postalCode : ''}
-                            <br/>
-                            Canada
-                        </td>
-                    </tr>    
-                    </tbody></table>                    
-
-                    <p style={{ textAlign:'right' }}>
-                        ............................................................................................................
-                        <br/>
-                        <i>Respondent/Solicitor for the Respondent</i>
-                    </p>
-
-                </div>                                                                                                    
-            );
-        } else {
-            return null;
-        }
+        return ( 
+            <div dangerouslySetInnerHTML={{ __html: this.props.content }} />                  
```

Comment:
Danger! Will Robinson, Danger!

I would really like it if we could avoid using dangerouslySetInnerHTML ... it is a red flag that we're going down the wrong path...

Reply:
I'm hoping there's a way to avoid it.  I understand why you did it.... thinking about it now.